component/Navigation/index.js

```diff
@@ -0,0 +1,23 @@
+import {
+  StyledLink,
+  StyledNavIcon,
+  StyledNavList,
+  StyledNavText,
+} from "./Navigation.styled";
+
+export default function Navigation() {
+  return (
+    <nav>
+      <StyledNavList>
+        <StyledLink href="/">
+          <StyledNavIcon>🏠</StyledNavIcon>
+          <StyledNavText>Home</StyledNavText>
+        </StyledLink>
```

Comment:
the direct children of your unordered list should be list items. Then your HTML is semantically perfect 🙌 

```suggestion
      <li>
        <StyledLink href="/">
          <StyledNavIcon>🏠</StyledNavIcon>
          <StyledNavText>Home</StyledNavText>
        </StyledLink>
        </li>
```